src/BigQuery/BigQueryClient.php

```diff
@@ -182,6 +272,10 @@ public function runQuery($query, array $options = [])
      * @param array $options [optional] {
      *     Configuration options.
      *
+     *     @type array $parameters Only available for standard SQL queries.
```

Comment:
I think a `{@see Google\Cloud\BigQuery\BigQueryClient::runQuery}` ref would be useful. Link back to where people can see examples of parameters in a query. Or add examples using parameters to this method as well.